lib/Local/LocalWebDriver.php

```diff
@@ -38,8 +33,6 @@ public static function create(
     /**
      * @param string $session_id
      * @param string $selenium_server_url
-     * @param null $connection_timeout_in_ms
-     * @param null $request_timeout_in_ms
```

Comment:
why are they removed?

Reply:
Php-cs-fixer thinks they are superfluous, which - in fact - are. However this conflicts with other check, so I deleted them altogether (they should be inherited from the parent class).

Reply:
okay